### DISCUSSION TITLE
[IOS] Nightguard alerts duplicate notifications

### DISCUSSION THREAD

**Discussion by ruifcnunes** [NightscoutFoundation/xDrip#3828] [Q&A] [answered]
Hey, 
Noticed that on iOS, the app always sends duplicate notifications for glucose alerts. The second one comes within 2-3 seconds from the first.
Is this normal? Is there a way to disable this?

 Thanks in advance

**Comment by Navid200** [comment, marked as answer]
xDrip only runs on Android.
The project that provides an app for iOS was copied from xDrip at some point years ago and converted to iOS.  But, other than that, there is no other associations between the two projects.  And for all I know, the two projects have diverged.  There is no tracking between the two.

**Reply by ruifcnunes** [reply]
ah, got it. Thank you for clarifying!
